x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/LocalStateCompositeXPackPlugin.java

```diff
@@ -476,4 +482,13 @@ public void close() throws IOException {
                 .collect(Collectors.toList());
     }
 
+    private Environment getUpdatedEnvironment(Environment existingEnvironment){
```

Comment:
Could this method simply return `existingEnvironment` if `inFipsSunJsseJvm()` returns false, and avoid creating a new environment object at all?

Reply:
yes, I like this idea - thanks !